internal/providers/terraform/aws/testdata/s3_bucket_test/s3_bucket_test.golden

```diff
@@ -2,70 +2,14 @@
- │  ├─ Retrieval requests (standard)        Monthly cost depends on usage: $0.03 per 1k requests    
- │  ├─ Retrievals (standard)                Monthly cost depends on usage: $0.01 per GB             
- │  ├─ Select data scanned (standard)       Monthly cost depends on usage: $0.008 per GB            
- │  ├─ Select data returned (standard)      Monthly cost depends on usage: $0.01 per GB             
- │  ├─ Retrieval requests (expedited)       Monthly cost depends on usage: $10.00 per 1k requests   
- │  ├─ Retrievals (expedited)               Monthly cost depends on usage: $0.03 per GB             
- │  ├─ Select data scanned (expedited)      Monthly cost depends on usage: $0.02 per GB             
- │  ├─ Select data returned (expedited)     Monthly cost depends on usage: $0.03 per GB             
- │  ├─ Select data scanned (bulk)           Monthly cost depends on usage: $0.001 per GB            
- │  ├─ Select data returned (bulk)          Monthly cost depends on usage: $0.0025 per GB           
- │  └─ Early delete (within 90 days)        Monthly cost depends on usage: $0.0036 per GB           
- └─ Glacier deep archive                                                                            
-    ├─ Storage                              Monthly cost depends on usage: $0.00099 per GB          
-    ├─ PUT, COPY, POST, LIST requests       Monthly cost depends on usage: $0.05 per 1k requests    
+ └─ Standard                                                                                        
+    ├─ Storage                              Monthly cost depends on usage: $0.023 per GB            
+    ├─ PUT, COPY, POST, LIST requests       Monthly cost depends on usage: $0.005 per 1k requests   
     ├─ GET, SELECT, and all other requests  Monthly cost depends on usage: $0.0004 per 1k requests  
-    ├─ Lifecycle transition                 Monthly cost depends on usage: $0.05 per 1k requests    
-    ├─ Retrieval requests (standard)        Monthly cost depends on usage: $0.10 per 1k requests    
-    ├─ Retrievals (standard)                Monthly cost depends on usage: $0.02 per GB             
-    ├─ Retrieval requests (bulk)            Monthly cost depends on usage: $0.025 per 1k requests   
-    ├─ Retrievals (bulk)                    Monthly cost depends on usage: $0.0025 per GB           
-    └─ Early delete (within 180 days)       Monthly cost depends on usage: $0.00099 per GB          
+    ├─ Select data scanned                  Monthly cost depends on usage: $0.002 per GB            
+    └─ Select data returned                 Monthly cost depends on usage: $0.0007 per GB           
                                                                                                     
  aws_s3_bucket.bucket_withUsage                                                                     
- ├─ Object tagging                                        0.1  10k tags                       $0.00 
```

Comment:
@aliscott You were right, the default behavior is to use either usage params or lifecycle options, so I haven't changed anything in the logic and it just works™. However, the Object tagging cost component relies on `lifecycle_rule` blocks, so it's not shown even when the usage param is set.